net/http/authz/authorizer.go

```diff
@@ -16,19 +16,28 @@ import (
 
 var ErrNotAuthorized = errors.New("not authorized")
 
+var builtinGrants []*Grant // initialized in loopback_authz.go
+
 type Authorizer struct {
 	raftDB        *raft.Service
 	raftPrefix    string
 	policyByRoute map[string][]string
+	extraGrants   map[string][]*Grant
```

Comment:
(Actually, I might have called it just `extra` since the fact that it holds grants is also obvious from context.)